include/novas.h

```diff
@@ -19,11 +19,16 @@
 #ifndef _NOVAS_
 #define _NOVAS_
 
-#include <math.h>   // for M_PI
+#include <math.h>   // for sin, cos
 #include <stdlib.h> // NULL
 #include <stdint.h>
 #include <time.h>
 
+/// Definition of π in case it's not defined in math.h
```

Comment:
Haha, github being too smart here, and replaced \&pi; with &pi. In the doc you want the former. Mea culpa.